### ISSUE TITLE
build.rs checks for CARGO_RUSTC_WRAPPER which does not seem to be a thing

### ISSUE THREAD

**Issue by RalfJung** [dtolnay/anyhow#248]
There is an env var called `RUSTC_WRAPPER`, but the build script looks for `CARGO_RUSTC_WRAPPER` which I cannot find anywhere in the cargo docs or source. Is that a typo?

https://github.com/dtolnay/anyhow/blob/db32585e16729bbdb798ba71ac30515127c9faee/build.rs#L74

**Comment by dtolnay** [comment]
This was added by @jonhoo in #157.

I guess it used to be a thing?

```console
$ git log -S CARGO_RUSTC_WRAPPER

commit 715fab4ff508d96fdbcc7d1b89e562397b5584a4
Author: Jon Gjengset <jongje@amazon.com>
Date:   Fri Jun 25 09:23:32 2021 -0700

    Update docs

commit 1e407453c2e382ff94c06bb03ab7336c8e1efd95
Author: Jon Gjengset <jongje@amazon.com>
Date:   Thu Jun 24 09:55:57 2021 -0700

    Remove CARGO_ prefix for new build script envvars

commit 8012d328ee1559ce5ddf519e7d0611fddb20d739
Author: Jon Gjengset <jongje@amazon.com>
Date:   Fri Jun 18 12:40:29 2021 -0700

    Avoid coercing path to string for windows

commit 719eb593c8e167b07caf33a2b23bddd085d3b1d9
Author: Jon Gjengset <jongje@amazon.com>
Date:   Fri Jun 18 12:14:58 2021 -0700

    Add tests for rustc context build script env vars

commit 44491ee8a64c0a861315bb6986e4345398d917e0
Author: Jon Gjengset <jongje@amazon.com>
Date:   Fri Jun 18 11:59:06 2021 -0700

    Document build script envvars for rustc context

commit 6ee606be69c7175a3974d3ecbf47f2b44f003047
Author: Jon Gjengset <jongje@amazon.com>
Date:   Fri Jun 18 11:49:12 2021 -0700

    Inform build scripts of rustc compiler context
    
    Fixes #9600.
```

**Comment by RalfJung** [comment]
Yes it got [removed during PR discussion](https://github.com/rust-lang/cargo/pull/9601#issuecomment-867801269) and looks like that was never reflected into the anyhow PR.
